src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TemplatesPanel/TemplateSelect/TemplateSelectOption.jsx

```diff
@@ -32,7 +32,7 @@ export function TemplateSelectOption(props) {
             </Tag>
           ) : null}
           <Tag className="templates-option--field-count">
-            {fields.filter(f => !f.hide).length}
+            {fields.filter(f => !f.hide).length - 2}
```

Comment:
Can you comment this so in the future we know why it is `- 2`

Reply:
Updated in 7d3cc53f2b9cd60b573bcb4fa5dcff8fbf6229f5